libs/comments/website/src/lib/comment/comment.tsx

```diff
@@ -2,24 +2,80 @@ import {css} from '@emotion/react'
 import {styled} from '@mui/material'
 import {CommentAuthorType} from '@wepublish/website/api'
 import {BuilderCommentProps, useWebsiteBuilder} from '@wepublish/website/builder'
+import {useLayoutEffect, useState} from 'react'
 import {MdPerson, MdVerified} from 'react-icons/md'
 
+const urlRegex = /^(https?:\/\/)?([\da-z.-]+)\.([a-z.]{2,6})([/\w .-]*)*\/?$/
+
+const isLink = (flair: string) => {
```

Comment:
Probably shouldn't be part of website builder.

Reply:
removed